From 4ddda342b379fe16efaea7cb206f560f34a6129d Mon Sep 17 00:00:00 2001
From: Nick Morrott <knowledgejunkie@gmail.com>
Date: Thu, 31 Dec 2020 00:44:11 +0000
Subject: [PATCH 34/50] dk_dr: disable grabber after source site disappeared.

A new source site [1] with a JSON API was mentioned in the bug report
(see below). This may be able to provide a replacement service, terms
and conditions allowing.

  [1] https://www.dr-massive.com/drtv/tv-guide

Refs: #119
---
 Makefile.PL | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/Makefile.PL b/Makefile.PL
index 270add4c..c85c976e 100644
--- a/Makefile.PL
+++ b/Makefile.PL
@@ -291,12 +291,12 @@ my @opt_components
                     'URI::URL'          => 0, },
      },
 
-     { name => 'tv_grab_dk_dr',
-       blurb => 'Grabber for Denmark (dr.dk)',
-       exes => [ 'grab/dk_dr/tv_grab_dk_dr' ],
-       prereqs => { 'DateTime'       => 0,
-                    'IO::Scalar'     => 0, },
-     },
+     # { name => 'tv_grab_dk_dr',
+     #   blurb => 'Grabber for Denmark (dr.dk)',
+     #   exes => [ 'grab/dk_dr/tv_grab_dk_dr' ],
+     #   prereqs => { 'DateTime'       => 0,
+     #                'IO::Scalar'     => 0, },
+     # },
 
      { name => 'tv_grab_eu_epgdata',
        blurb => '$$ Grabber for some European countries (epgdata.com)',
-- 
2.29.2

